### ISSUE TITLE
bundle() having trouble with path containing spaces?

### ISSUE THREAD

**Issue by OClement** [systemjs/builder#486]
I'm having issues in trying to create multiple bundles through a gulp task
Because of an apparent limitation with gulp, I want to pass an absolute path to the bundler, but it complains about it:

```
Unhandled rejection TypeError: Expected operator before AFTER_SPACE\OF\PATH\TO\PROJECT_ROOT\EntyPoint.js
    at parseExpression (C:\PROJECT\BEFORE_SPACE AFTER_SPACE\OF\PATH\TO\PROJECT_ROOT\node_modules\systemjs-builder\lib\arithmetic.js:24:13)
    at exports.traceExpression (C:\PROJECT\BEFORE_SPACE AFTER_SPACE\OF\PATH\TO\PROJECT_ROOT\node_modules\systemjs-builder\lib\arithmetic.js:139:20)
    at C:\PROJECT\BEFORE_SPACE AFTER_SPACE\OF\PATH\TO\PROJECT_ROOT\node_modules\systemjs-builder\lib\builder.js:545:12
    at tryCatcher (C:\PROJECT\BEFORE_SPACE AFTER_SPACE\OF\PATH\TO\PROJECT_ROOT\node_modules\systemjs-builder\node_modules\bluebird\js\release\util.js:11:23)
    at Promise._settlePromiseFromHandler (C:\PROJECT\BEFORE_SPACE AFTER_SPACE\OF\PATH\TO\PROJECT_ROOT\node_modules\systemjs-builder\node_modules\bluebird\js\release\promise.js:489
:31)
    at Promise._settlePromise (C:\PROJECT\BEFORE_SPACE AFTER_SPACE\OF\PATH\TO\PROJECT_ROOT\node_modules\systemjs-builder\node_modules\bluebird\js\release\promise.js:546:18)
    at Promise._settlePromiseCtx (C:\PROJECT\BEFORE_SPACE AFTER_SPACE\OF\PATH\TO\PROJECT_ROOT\node_modules\systemjs-builder\node_modules\bluebird\js\release\promise.js:583:10)
    at Async._drainQueue (C:\PROJECT\BEFORE_SPACE AFTER_SPACE\OF\PATH\TO\PROJECT_ROOT\node_modules\systemjs-builder\node_modules\bluebird\js\release\async.js:130:12)
    at Async._drainQueues (C:\PROJECT\BEFORE_SPACE AFTER_SPACE\OF\PATH\TO\PROJECT_ROOT\node_modules\systemjs-builder\node_modules\bluebird\js\release\async.js:135:10)
    at Immediate.Async.drainQueues [as _onImmediate] (C:\PROJECT\BEFORE_SPACE AFTER_SPACE\OF\PATH\TO\PROJECT_ROOT\node_modules\systemjs-builder\node_modules\bluebird\js\release\as
ync.js:16:14)
    at processImmediate [as _immediateCallback] (timers.js:384:17)
```

---

I would rather use a relative path but gulp doesn't use the proper working directory;

I will probably reorganize my code as a work around but thought I would create this issue in the meantime

**Comment by guybedford** [comment]
This is actually a SystemJS builder bug that `bundle('/some path/asdf')` will throw like that. We can do some polishing around these use cases.

But yes, setting the baseURL properly is important for builds.

**Comment by guybedford** [comment]
Fixed in https://github.com/systemjs/builder/commit/37bcdb2603720c4ef026e4cd982fd70a16cf75b2.

**Comment by guybedford** [comment]
Released in 0.15.8.
